.circleci/docker/common/install_llvm.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+llvm_url="https://github.com/llvm/llvm-project/releases/download/llvmorg-9.0.1/llvm-9.0.1.src.tar.xz"
+
+
+pushd /tmp
+wget --no-verbose --output-document=llvm.tar.xz "$llvm_url"
+mkdir llvm
+tar -xf llvm.tar.xz -C llvm --strip-components 1
+rm -f llvm.tar.xz
+
+cd llvm
+mkdir build
+cd build
+cmake -G "Unix Makefiles" \
+  -DCMAKE_BUILD_TYPE=MinSizeRel \
```

Comment:
out of curiosity which option disables tests, otherwise i think we get linking issues if something else links in gtest?

Reply:
I haven't had any linking issues with test?